specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2021-10-01/costmanagement.json

```diff
@@ -1052,6 +1069,9 @@
               "$ref": "common-types.json#/definitions/ErrorResponse"
             }
           }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
I'm confused about why is nextLinkName null? Did you not declare a nextLink property?

Reply:
@TimLovellSmith  `QueryProperties` starting on line 2283 already has a nextLink property. I essentially copied what was in `dimensions`  (870-949) for the pageable value there, which led to the `"nextLinkName": null` decision, but it looks like this should be `"nextLinkname": "nextLink"` instead... Does this seem correct? If so I can commit the change.

Reply:
Yeah, I think so.

>Required. Specifies the name of the property that provides the next link (common: nextLink). If the model does not have a next link property then specify null. This is useful for services that return an object that has an array referenced by itemName. The object is then flattened in a way that the array is directly returned, no paging is used. This provides a better client side API to the end user.

Reply:
You probably want to fix the other one too, especially if your service returns it today. It would be a spec bugfix breaking change that should be able to be approved as a bugfix.

Reply:
@TimLovellSmith thanks. You'll see 2 additional commits because I also had to sync main (hopefully this doesn't cause any issues). I've changed the `"nextLinkName": null` in both places.